backend/test/post_comment_handler_test.py

```diff
@@ -32,44 +32,79 @@ def setUp(cls):
              ("/api/posts/(.*)/comments", PostCommentHandler)
              ], debug=True)
         cls.testapp = cls.webtest.TestApp(app)
-        initModels(cls)
+        
+        # Create models
+        # new User
+        cls.user = mocks.create_user(USER_EMAIL)
+        # new User
+        cls.other_user = mocks.create_user(OTHER_USER_EMAIL)
+        # new Institution
+        cls.institution = mocks.create_institution()
+        cls.institution.members = [cls.user.key]
+        cls.institution.followers = [cls.user.key]
+        cls.institution.admin = cls.user.key
+        cls.institution.put()
+        # POST of user To institution 
+        cls.user_post = mocks.create_post(cls.user.key, cls.institution.key)
+        # Comments
+        cls.comment = {'text': 'Frist comment. Using in Test', 'institution_key': cls.institution.key.urlsafe()}
+        cls.other_comment = {'text': 'Second comment. Using in Test', 'institution_key': cls.institution.key.urlsafe()}
+        # http post body
+        cls.body = {
+            'commentData': cls.comment,
+            'currentInstitution': {
+                'name': cls.institution.name
+            }
+        }
 
-    @patch('utils.verify_token', return_value={'email': MAIANA_EMAIL})
-    def test_post(self, verify_token):
-        """Another user comment in Post of Mayza."""
+    @patch('handlers.post_comment_handler.enqueue_task')
+    @patch('utils.verify_token', return_value={'email': OTHER_USER_EMAIL})
+    def test_post(self, verify_token, enqueue_task):
+        """Another user comment in Post of user."""
```

Comment:
Why another? If it's repeated maybe it's not necessary.

Reply:
On test_post, i am testing the case a user, that is not the post author, comment on the post.
On test_post_ownerpost, i am testing the case in which the post's author comments on its own post.